pkg/visitor/content_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package visitor
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/apigee/registry/pkg/connection/grpctest"
+	"github.com/apigee/registry/pkg/names"
+
+	"github.com/apigee/registry/rpc"
+	"github.com/apigee/registry/server/registry/test/seeder"
+)
+
+func TestContentHelpers(t *testing.T) {
+	specName, err := names.ParseSpec("projects/content-test/locations/global/apis/a/versions/v/specs/s")
+	if err != nil {
+		t.Fatalf("failed to parse spec name %s", specName)
+	}
```

Comment:
Parsing and checking the error is unnecessary. We know what name we want, and the error should never happen.
```
specName := names.Spec{
    ProjectID: "content-test",
    ApiID: "a",
    VersionID: "v",
    SpecID: "s",
}
```

Reply:
thanks - along those lines, I think it would be nice to pull a pattern from [regexp](https://pkg.go.dev/regexp#MustCompile) and be able to do this in one line (in tests):
```
specName := names.MustParseSpec("projects/content-test/locations/global/apis/a/versions/v/specs/s")
```